src/lib/index.jsx

```diff
@@ -24,6 +26,8 @@ type State = {
   otp: string[],
 };
 
+const isStyleObject = obj => typeof obj === 'object';
```

Comment:
Write a comment mentioning drawbacks, e.g. "Doesn't really check if it's a style object, if there is, please open a PR" something like this